adm/style/acp_phpbb_ideas.html

```diff
@@ -14,30 +14,42 @@
 <!-- ENDIF -->
 
 <form id="acp_phpbb_ideas_settings" method="post" action="{U_ACTION}">
-
-<!-- BEGIN options -->
-	<!-- IF options.S_LEGEND -->
-		<!-- IF not options.S_FIRST_ROW -->
-			</fieldset>
-		<!-- ENDIF -->
-		<fieldset>
-			<legend>{options.LEGEND}</legend>
-	<!-- ELSE -->
+	<fieldset>
+		<legend>{L_ACP_PHPBB_IDEAS_SETTINGS}</legend>
 		<dl>
-			<dt><label for="{options.KEY}">{options.TITLE}{L_COLON}</label><!-- IF options.S_EXPLAIN --><br /><span>{options.TITLE_EXPLAIN}</span><!-- ENDIF --></dt>
-			<dd>{options.CONTENT}</dd>
-			<!-- IF options.KEY == 'ideas_poster_id' --><dd>[ <a href="{U_FIND_USERNAME}" onclick="find_username(this.href); return false;">{L_FIND_USERNAME}</a> ]</dd><!-- ENDIF -->
+			<dt><label for="ideas_forum_id">{L_ACP_IDEAS_FORUM_ID}{L_COLON}</label><br /><span>{L_ACP_IDEAS_FORUM_ID_EXPLAIN}</span></dt>
+			<dd>{S_FORUM_SELECT_BOX}</dd>
+		</dl>
+		<dl>
+			<dt><label for="ideas_poster_id">{L_ACP_IDEAS_POSTER_ID}{L_COLON}</label><br /><span>{L_ACP_IDEAS_POSTER_ID_EXPLAIN}</span></dt>
+			<dd><input id="ideas_poster_id" type="text" size="45" maxlength="255" name="config[ideas_poster_id]" value="{IDEAS_POSTER}" /></dd>
+			<dd>[ <a href="{U_FIND_USERNAME}" onclick="find_username(this.href); return false;">{L_FIND_USERNAME}</a> ]</dd>
+		</dl>
+		<dl>
+			<dt><label for="ideas_base_url">{L_ACP_IDEAS_BASE_URL}{L_COLON}</label><br /><span>{L_ACP_IDEAS_BASE_URL_EXPLAIN}</span></dt>
+			<dd><input id="ideas_base_url" type="text" size="45" maxlength="255" name="config[ideas_base_url]" value="{IDEAS_BASE_URL}" /></dd>
 		</dl>
-	<!-- ENDIF -->
-<!-- END options -->
-</fieldset>
-
-<fieldset class="submit-buttons">
-	<input class="button1" type="submit" id="submit" name="submit" value="{L_SUBMIT}" />&nbsp;
-	<input class="button2" type="reset" id="reset" name="reset" value="{L_RESET}" />
-	{S_FORM_TOKEN}
-</fieldset>
 
+		<fieldset class="submit-buttons">
+			<input class="button1" type="submit" id="submit" name="submit" value="{L_SUBMIT}" />&nbsp;
+			<input class="button2" type="reset" id="reset" name="reset" value="{L_RESET}" />
+			{S_FORM_TOKEN}
+		</fieldset>
+	</fieldset>
 </form>
 
+<!-- IF S_IDEAS_FORUM_ID -->
```

Comment:
Another thought on this. Instead of hiding this, we could instead add this to the button instead:
`<!-- IF S_IDEAS_FORUM_ID -->disabled<!-- ENDIF -->`
This would disable the button instead of hiding the whole ting. It may be better.